### PR TITLE
update project presentation to new readme of deliverables format

### PR DESCRIPTION
**Pull request by JessicaS11**
Updates r.e. projects to add a readme template and update project pages around using it

### CODE REVIEW

File: book/projects/list_of_projects.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # List of Projects
 
 Here is where we will be listing the projects during the {{dates}} {{ hackweek }} hackweek:
+[Project Jupyterbook]().
```
Comment (JessicaS11) [inline book/projects/list_of_projects.md]: We still need to fill this link in.

File: book/projects/project_roadmap.md [hunk 2]
```diff
@@ -44,10 +44,11 @@ We facilitate a project team formation session at the end of the first day of th
 * project work begins and teams have regular check-ins to chart progress
 * project outputs are recorded in a GitHub repository
 
-## **Presentations** 
+## **Deliverables** 
+
+* each team is requested to create a project readme using [this template](project_readmes/project_readme_template) and contribute it to the this Jupyterbook. Please check in with a helper if your team would like guidance through this process. The basic workflow will require creating a branch, copying the template file, filling in the template file and adding it to the list of projects, committing your changes, opening a pull request (PR), and reviewing and merging your changes.
```
Comment (jomey) [inline book/projects/project_roadmap.md]: For consistency, want to capitalize the beginning of each bullet point?

**Review by jomey — APPROVED** [review]

File: book/projects/list_of_projects.md [hunk 3]
```diff
@@ -4,4 +4,4 @@ Here is where we will be listing the projects during the {{dates}} {{ hackweek }
 
 | Project Name (with link to GitHub repo) | slack channel | final presentation | 
 |:--------|:--------|:-----|
-| [Snow Temperature Changes](https://github.com/snowex-hackweek/hot-pow) | [#Hot-Pow](https://snowexhackweek2021.slack.com/archives/C027RBZ9JH3) | [Video](https://youtu.be/lV3to0c_KtA)  |
\ No newline at end of file
+| [Snow Temperature Changes](https://github.com/snowex-hackweek/hot-pow) | [#Hot-Pow](https://snowexhackweek2021.slack.com/archives/C027RBZ9JH3) | [README](project_readmes/project_readme_template)  |
```
Comment (jomey) [inline book/projects/list_of_projects.md]: Maybe use a past ICESat-2 project to be less cross-hackweek linking?

File: book/projects/project_roadmap.md (hunk 2 above)
Comment (JessicaS11) [reply book/projects/project_roadmap.md]: Consistency above (lower case, clearly the one I looked at) or below (upper case)?

**Review by scottyhq — APPROVED** [review]
Looks great to me, Could you also fix the project github repository template links on this page?  https://deploy-preview-76--icesat-2-2023.netlify.app/projects/project_initialization#use-our-github-project-template

**Comment by JessicaS11** [comment]
> 



> Looks great to me, Could you also fix the project github repository template links on this page? https://deploy-preview-76--icesat-2-2023.netlify.app/projects/project_initialization#use-our-github-project-template

Any idea why they're not rendering correctly? Should I just hard code them?

File: book/projects/project_roadmap.md (hunk 2 above)
Comment (jomey) [reply book/projects/project_roadmap.md]: My vote is upper case :shrug:

**Comment by scottyhq** [comment]
I don't know what the issue is with link substitutions, I remember issues coming up int he past with that. I'd just use standard markdown links `[link description](url)` and we can open an issue to tackle the substitutions later

File: book/projects/project_initialization.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ Before getting started, you'll need a name for your project. A team exploring sn
 We have created a GitHub template that is pre-populated with directories and a README layout the we find works well with most teams.
 
 We encourage all teams to create a new GitHub repository within the {{hackweek}} Organization, even if it is just to point to an already-existing repository or package your team is contributing to for their project.
-1. Go to: {{ '[https://github.com/`{hw}`](https://github.com/{hw})'.format(hw=hackweek) }}
+1. Go to: {{ '[`{github_org_url}`]({github_org_url})'.format(hw=hackweek) }}
```
Comment (jomey) [inline book/projects/project_initialization.md]: Looks like this is causing trouble.
If link name and target are the same, you could try this syntax:
https://www.markdownguide.org/basic-syntax#urls-and-email-addresses

Comment (JessicaS11) [reply book/projects/project_initialization.md]: It was plugging in "hw", which is the event name and has a space in it (and thus is an invalid url). I changed it to use `github_org_url`. Must have missed this when we previously separated out the link vs name in the config.

Comment (JessicaS11) [reply book/projects/project_initialization.md]: So we should flag this to fix in the template - do we have a spot going for this here or in the template?

Comment (jomey) [reply book/projects/project_initialization.md]: There is also a pending update to JupyterBook we haven't done (which might improve handling of string interpolation) and I suggest to log an issue

**Pull request merged**